src/i18n.ts

```diff
@@ -138,7 +138,10 @@ i18n
             createdPrescription: 'An error occurred while creating the prescription',
             userLogin: 'Invalid username or password. Please try again',
             errorRegister: 'Error during registration',
-            userUpdated: 'Error updating profile'
+            userUpdated: 'Error updating profile',
+            requiredField: 'This field is required',
+            usernameMinLengthError: 'You must enter at least 2 characters.',
+            passwordMinLengthError: 'You must enter at least 6 characters.'
```

Comment:
Can't we use a constant that we reference to?

Reply:
No possible

Reply:
Why not? Could you elaborate

Reply:
done good advise